data/yam/agama/auto/opensuse_default.jsonnet

```diff
@@ -1,20 +1,27 @@
 {
-  "product": {
-    "id": "Leap_16.0"
+  product: {
+    id: {{PRODUCT}}
   },
-  "software": {
```

Comment:
and software? this is not default.

Reply:
No such 'software' section in original profile. I added it to this file, do we need to add this for all default profiles?